apps/namadillo/src/atoms/chain/atoms.ts

```diff
@@ -48,7 +38,19 @@ export const chainAtom = atomWithQuery<ChainSettings>((get) => {
         nativeTokenAddress: chainParameters.data!.nativeTokenAddress,
         checksums: chainParameters.data!.checksums,
       };
-    }, [!!indexerUrl, indexerRpcUrl, chainParameters, tomlConfig]),
```

Comment:
Please, review here carefully

I removed the `indexerRpcUrl` and `tomlConfig` because they don't have direct relation with the `queryFn`, meaning they are useless for it

If it's used by a child request, we should add the `queryDependentFn` on this child request to handle this case

Or if it's another edge case, we should add a comment for future developers